root/etc/e-smith/db/configuration/migrate/Remove_logs_delegation

```diff
@@ -0,0 +1,18 @@
+{
+    #
+    # Remove the `logs` delegation to users  
+    #
+    use esmith::ConfigDB;
+    
+    my $db = esmith::ConfigDB->open();
+    my $newProp = '';
+    my @Keys = split(',', $db->get_prop('cockpit.socket','delegation')|| '');
+
+    foreach (@Keys) {
+        my @props = split(':',$_);
```

Comment:
Skip group name (first element of the list)
```suggestion
        my ($group_name, @props) = split(':',$_);
```

Reply:
done